__tests__/app.test.js

```diff
@@ -67,7 +67,37 @@ describe("app!!", ()=>{
           expect(sortedArticlesKeys).toEqual(expect.arrayContaining(['article_id', 'comment_count']))
         })
       })
+      //            TASK 6
+        test('should return comments if there is any',()=>{
+          return request(app)
+          .get('/api/articles/1/comments')
+          .expect(200)
+          .then((response)=>{
+            const comments = response.body.comments
+            //dynamic expected needed (flexible test)
+            //iterate through the input comments with article_id1
+            for(let i =0; i< comments.length; i++){
+              expect(comments[i]).toHaveProperty('article_id', 1)
+            }
+          })
+        })
+        //does not console.log(err)
+        test('should return empty arr when input has no comments',()=>{
+          return request(app)
+          .get('/api/articles/4/comments')
+          .expect(200)
+          .then((response)=>{
+            expect(response.body.comments).toEqual([])
+          })
+        })
+        //check if the articles exist
+        test('should return err when input is wrong',()=>{
+          return request(app)
+          .get('/api/articles/123989/comments')
+          .expect(404)
+          .then((response)=>{
+            expect(response.body).toEqual({"message": "Article does not exist!"})
+          })
```

Comment:
Good tests here, great that you've checked you get a 200 if there is a valid article with no comments
Could you add a test that checks a 400 is returned if you try and find the comments for an article with an invalid id, like `/api/articles/katherine/comments` for example